backend/src/api/app_factory.py

```
@@ -187,29 +199,41 @@ async def enhanced_startup_event():
-            logger.info(f"Service health check: {healthy_services}/{total_services} services healthy")
+            logger.info(
+                f"Service health check: {healthy_services}/{total_services} "
+                f"services healthy"
+            )
 
             # Start legacy cleanup service for backward compatibility
             try:
                 cleanup_service = container.get_cleanup_service()
                 asyncio.create_task(cleanup_service.start_periodic_cleanup(
                     interval_hours=settings.cleanup_interval_hours
                 ))
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)